packages/tabs/src/useTabs.js

```diff
@@ -0,0 +1,55 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import { useSelection } from '@zendeskgarden/container-selection';
+
+export function useTabs({ vertical, ...options } = {}) {
+  const { selectedItem, focusedItem, getContainerProps, getItemProps } = useSelection({
+    direction: vertical ? 'vertical' : 'horizontal',
+    ...options
+  });
+
+  const getTabListProps = ({ role = 'tablist', ...other } = {}) => {
+    return {
+      role,
+      ...other
+    };
+  };
+
+  const getTabProps = ({ role = 'tab', key, ...other } = {}) => {
+    return {
+      key,
+      role,
+      ...other
+    };
+  };
+
+  const getTabPanelProps = ({ role = 'tabpanel', key, ...other } = {}) => {
```

Comment:
`key` should no longer be necessary in this implementation

Reply:
We need `key` here to be able to check it against the `selectedItem` to know when to flip the `aria-hidden` attribute

Reply:
I've got some more a11y stuff I need to do for this so I think I can solve it a better way here as it's not obvious you need to apply the same key to this as the `getTabProps` setter.

Reply:
👍 Ah cool. IMO using the `key` naming was a mistake when I originally did it. Get's really confusing why it's necessary outside of a `.map()`.

I'll reserve feedback till it's ready 🎂